Sources/PluginInterface/XCodeTarget.swift

```diff
@@ -20,14 +20,24 @@ public protocol XCodeTarget {
     /// `.h` & `.pch`
     /// headers in current Target(Bazel Package)
     var headers: [String] { get }
+    /// `.hpp`
+    /// headers in current Target(Bazel Package)
+    var hpps: [String] { get }
     
-    /// swift
-    /// m
-    /// mm
-    /// ...
+    /// all build file
     var srcs: [String] { get }
-    /// var srcs_swift: [String] { get }
-    /// var hdr
+    /// `.c`
+    var srcs_c: [String] { get }
+    /// `.m`
+    var srcs_objc: [String] { get }
+    /// `.cpp`
```

Comment:
Maybe remove these comments because the getter name says the same thing?

https://quotepark.com/quotes/1944473-martin-fowler-when-you-feel-the-need-to-write-a-comment-first-t/